docs/servo_power_test/src/config.ts

```diff
@@ -0,0 +1,107 @@
+import {Graph} from './graph';
+import {AnnotationData, PowerData} from './power_test_controller';
+import {ServoController} from './servo_controller';
+import {TestRunner} from './test_runner';
+import {Ui} from './ui';
+
+export class Config {
+  private INTERVAL_MS = 100;
+  private ui: Ui;
+  private servoController: ServoController;
+  private runner: TestRunner;
+  public powerDataList: Array<PowerData> = [];
+  public annotationList: Array<AnnotationData> = [];
+  public graph: Graph;
+  public customScript: string;
+  public halt = true;
```

Comment:
do we have to make the halt flag public...? less public fields are generally better.
https://google.github.io/styleguide/tsguide.html#visibility